File: katas/8 kyu/convert-a-number-to-a-string/solution.py
# convert-a-number-to-a-string
import codewars_test as test
# -------------------------------- MY SOLUTION ------------------------------- #
number_to_string = lambda x : str(x)
# ------------------------------ CLEVER SOLUTION ----------------------------- #
number_to_string = str
# ----------------------------------- TEST ----------------------------------- #
test.assert_equals(number_to_string(67), '67')
test.assert_equals(number_to_string(79585), '79585')
test.assert_equals(number_to_string(79585), "79585")
test.assert_equals(number_to_string(1+2), '3')
test.assert_equals(number_to_string(1-2), '-1')

File: katas/8 kyu/convert-a-number-to-a-string/solution.js
// convert-a-number-to-a-string
/* ------------------------------- MY SOLUTION ------------------------------ */
function numberToString(num) {
  return num.toString()
}
/* ----------------------------- CLEVER SOLUTION ---------------------------- */
/* ---------------------------------- TEST ---------------------------------- */
// const chai = require("chai");
// const Test = chai.assert.strictEqual; // replace Test.assertEqual with Test
// chai.config.truncateThreshold=0;

const assert = require('chai').assert;
describe("Tests", () => {
  it("test", () => {
    assert.strictEqual(numberToString(67), '67');
  });
});
